manifests/f/Flywheel/Local/6.0.0/Flywheel.Local.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using YamlCreate.ps1 v1.1.5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Flywheel.Local
+PackageVersion: 6.0.0
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: nullsoft
+  Scope: user
+  InstallerUrl: https://cdn.localwp.com/releases-stable/6.0.0+5444/local-6.0.0-windows.exe
+  InstallerSha256: 9FE7DAF99B22C91B53785340F47678EF450B2DC485352CEBFBA9B51537EB1F6E
+  UpgradeBehavior: install
```

```suggestion
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: nullsoft
  Scope: user
  InstallerUrl: https://cdn.localwp.com/releases-stable/6.0.0+5444/local-6.0.0-windows.exe
  InstallerSha256: 9FE7DAF99B22C91B53785340F47678EF450B2DC485352CEBFBA9B51537EB1F6E
  InstallerSwitches:
    Custom: /CURRENTUSER
  UpgradeBehavior: install
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: nullsoft
  Scope: machine
  InstallerUrl: https://cdn.localwp.com/releases-stable/6.0.0+5444/local-6.0.0-windows.exe
  InstallerSha256: 9FE7DAF99B22C91B53785340F47678EF450B2DC485352CEBFBA9B51537EB1F6E
  InstallerSwitches:
    Custom: /ALLUSERS
  UpgradeBehavior: install
```
Needed for compatibility